aws/resource_aws_ecr_repository.go

```diff
@@ -32,6 +33,15 @@ func resourceAwsEcrRepository() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"image_tag_mutability": {
```

Comment:
This Terraform attribute is missing a `d.Set("image_tag_mutability", repository.ImageTagMutability)` during the `Read` function as noted by this test failure:

```
--- FAIL: TestAccAWSEcrRepository_basic (14.30s)
    testing.go:568: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.

        (map[string]string) {
        }


        (map[string]string) (len=1) {
         (string) (len=20) "image_tag_mutability": (string) (len=7) "MUTABLE"
        }
```